Cesium3DTilesContent/include/Cesium3DTilesContent/SubtreeAvailability.h

```diff
@@ -10,80 +11,425 @@ struct QuadtreeTileID;
+   */
   bool isTileAvailable(
       const CesiumGeometry::QuadtreeTileID& subtreeID,
       const CesiumGeometry::QuadtreeTileID& tileID) const noexcept;
 
+  /**
+   * @brief Determines if a given tile in the subtree is available.
+   *
+   * @param subtreeID The ID of the root tile of the subtree.
+   * @param tileID The ID of the tile to query.
+   * @return True if the tile is available; otherwise, false.
+   */
   bool isTileAvailable(
       const CesiumGeometry::OctreeTileID& subtreeID,
       const CesiumGeometry::OctreeTileID& tileID) const noexcept;
 
+  /**
+   * @brief Determines if a given tile in the subtree is available.
+   *
+   * @param relativeTileLevel The level of the tile to query, relative to the
+   * root of the subtree.
+   * @param relativeTileMortonId The Morton ID of the tile to query. See
+   * {@link ImplicitTilingUtilities::computeRelativeMortonIndex}.
+   * @return True if the tile is available; otherwise, false.
+   */
   bool isTileAvailable(
       uint32_t relativeTileLevel,
       uint64_t relativeTileMortonId) const noexcept;
 
+  /**
+   * @brief Sets the availability state of a given tile in the subtree.
+   *
+   * @param subtreeID The ID of the root tile of the subtree.
+   * @param tileID The ID of the tile for which to set availability.
+   * @param isAvailable The new availability state for the tile.
+   */
+  void setTileAvailable(
+      const CesiumGeometry::QuadtreeTileID& subtreeID,
+      const CesiumGeometry::QuadtreeTileID& tileID,
+      bool isAvailable) noexcept;
+
+  /**
+   * @brief Sets the availability state of a given tile in the subtree.
+   *
+   * @param subtreeID The ID of the root tile of the subtree.
+   * @param tileID The ID of the tile for which to set availability.
+   * @param isAvailable The new availability state for the tile.
+   */
+  void setTileAvailable(
+      const CesiumGeometry::OctreeTileID& subtreeID,
+      const CesiumGeometry::OctreeTileID& tileID,
+      bool isAvailable) noexcept;
+
+  /**
+   * @brief Sets the availability state of a given tile in the subtree.
+   *
+   * @param relativeTileLevel The level of the tile for which to set
+   * availability, relative to the root of the subtree.
+   * @param relativeTileMortonId The Morton ID of the tile for which to set
+   * availability. See
+   * {@link ImplicitTilingUtilities::computeRelativeMortonIndex}.
+   * @param isAvailable The new availability state of the tile.
+   */
+  void setTileAvailable(
+      uint32_t relativeTileLevel,
+      uint64_t relativeTileMortonId,
+      bool isAvailable) noexcept;
+
+  /**
+   * @brief Determines if content for a given tile in the subtree is available.
+   *
+   * @param subtreeID The ID of the root tile of the subtree.
+   * @param tileID The ID of the tile to query.
+   * @return True if the tile's content is available; otherwise, false.
+   */
   bool isContentAvailable(
       const CesiumGeometry::QuadtreeTileID& subtreeID,
```

Comment:
If it's okay, I'll do a sweep to rename all the `subtreeID` and `tileID` variables to `subtreeId` and `tileId`, to be consistent with the capitalization of the other variable names.